src/content/packer/docs-landing.json

```diff
@@ -1,102 +1,74 @@
+	"pageSubtitle": "Packer lets you create identical machine images for multiple platforms from a single source configuration. A common use case is creating golden images for organizations to use in cloud infrastructure.",
 	"marketingContentBlocks": [
-		{
-			"type": "section-heading",
-			"title": "Use Cases"
-		},
 		{
 			"type": "card-grid",
-			"title": "Secrets Management",
-			"description": "Centrally store, access, and deploy secrets across applications, systems, and infrastructure.",
+			"title": "Build Images",
+			"description": "Standardize and automate your workflow.",
 			"cards": [
 				{
-					"description": "A generic Key-Value store used to store arbitrary secrets within the configured physical storage for Vault.",
-					"title": "KV Secrets Engine",
-					"url": "/vault/docs/secrets/kv"
+					"description": "Define the images you want Packer to build and the plugins Packer should use during the build process.",
+					"title": "Templates",
+					"url": "/packer/docs/templates"
 				},
 				{
-					"description": "Generate database credentials dynamically based on configured roles.",
-					"title": "Database Credentials",
-					"url": "/vault/docs/secrets/databases"
-				},
-				{
-					"description": "Deploy Vault into Kubernetes using the official HashiCorp Vault Helm chart.",
-					"title": "Kubernetes Secrets",
-					"url": "/vault/docs/platform/k8s"
+					"description": "Use the Packer CLI to build images, validate templates, and manage plugins.",
+					"title": "Packer CLI",
+					"url": "/packer/docs/commands"
 				}
 			]
 		},
 		{
 			"type": "card-grid",
-			"title": "Encryption Services",
-			"description": "Securely handle data such as social security numbers, credit card numbers, and other types of compliance-regulated information.",
+			"title": "Use Plugins",
+			"description": "Use either built-in or external plugins to perform tasks during each build.",
 			"cards": [
 				{
-					"description": "Apply cryptographic functions on data in-transit. Sign and verify data, generate hashes and HMACs of data, and act as a source of random bytes.",
-					"title": "Transit Secrets Engine",
-					"url": "/vault/docs/secrets/transit"
+					"description": "Builders create and configure virtual machines and then transform those virtual machines into a machine image.",
+					"title": "Builders",
+					"url": "/packer/docs/builders"
 				},
 				{
-					"description": "Handle secure data transformation and tokenization against a provided input value using NIST vetted cryptographic standards and other transformations such as masking.",
-					"title": "Transform Secrets Engine",
-					"url": "/vault/docs/secrets/transform"
+					"description": "Post-processors transform one artifact into another, like compressing files.",
```

Comment:
Could this be last since I think they run last in the build process and are optional?